osu.Game/Overlays/Changelog/ChangelogSupporterPromo.cs

```diff
@@ -92,8 +92,10 @@ public ChangelogSupporterPromo()
         }
 
         [BackgroundDependencyLoader]
-        private void load(OsuColour colour, TextureStore textures)
+        private void load(OsuColour colour, TextureStore textures, OverlayColourProvider colourProvider)
         {
+            background.Colour = colourProvider.Background5;
```

Comment:
You can probably just move the whole `InternalChildren = ...` block into  this method to avoid keeping a field.

Reply:
Yeah, sure.